pnp/PnpHelper.cs

```diff
@@ -49,7 +50,7 @@ public static Message CreateIothubMessageUtf8(string telemetryName, string seria
         /// <param name="propertyName">The property name, as defined in the DTDL interface.</param>
         /// <param name="serializedPropertyValue">The serialized property value, in the format defined in the DTDL interface.</param>
         /// <param name="componentName">(optional) The name of the component in which the property is defined. Can be null for property defined under the root interface.</param>
-        /// <returns>The property path for read-only and read-write property updates.</returns>
+        /// <returns>The property patch for read-only and read-write property updates.</returns>
```

Comment:
i had observed same typo in java but I think it is gone with my previous branch. Can you add to your existing java PR?

Reply:
It's updated in the java PR as well